docs/specs/meta-clients/web3modal/api.md

```diff
@@ -1,142 +1,248 @@
-# API
+# API (v2)
 
-Web3Modal has it's own api available at [api.web3modal.com](https://api.web3modal.com) that is used to fetch wallet data from a WalletConnect maintained [wallet & dapp registry](https://walletconnect.com/explorer).
+> For the deprecated v1 specification, see [here](./api-v1-legacy.md).
 
-All responses are cached for a minimum of 2h.
+## Query Parameters
+
+All API requests now use query parameters instead of custom headers for SDK metadata, making integration simpler and more compatible with common HTTP clients and browser caching mechanisms.
+
+### Mandatory Query Parameters
+
+All API endpoints expect the following mandatory query parameters:
+
+- `st=`: SDK type
+  - Options: `w3m | wcm`
+  - Example: `st=w3m`
+- `sf=`: SDK framework
+  - Options: `html | react | vue`
+  - Example: `sf=react`
```

Comment:
It'd be nice to include the platform as a query param, eg iOS or Android for clients like Flutter and React Native. This will allow deeper validation against bundle IDs (iOS) or package names (anroid).

Reply:
```
env=react-native@0.74.1
os=ios@17.4
```

Reply:
Added here as optional: https://github.com/WalletConnect/walletconnect-specs/pull/239/files#diff-6f25b86fe5ecc8d0a32393be4bab33e4f5b6ca64e95c9180dce7cf7ad76e88deR29

Assuming we don't want to make these mandatory for all possible clients but if we feel strongly about it please let me know 👍